src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+class Bulldozer extends Machine {

```

Comment:
```suggestion
public class Bulldozer extends Machine {

```
the same with other classes